pallets/ajuna-awesome-avatars/src/tests.rs

```diff
@@ -14,11 +14,105 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-use crate::mock::*;
+use crate::{mock::*, *};
+use frame_support::{assert_noop, assert_ok};
 
-#[test]
-fn always_good_to_test() {
-	new_test_ext().execute_with(|| {
-		assert_eq!(1 + 1, 2);
-	});
+#[cfg(test)]
+mod organizer {
+	use super::*;
+
+	const ALICE: u32 = 1;
+	const BOB: u32 = 2;
+	const CHARLIE: u32 = 3;
+	const DELTHEA: u32 = 4;
+	const ERIN: u32 = 5;
+	const FLORINA: u32 = 6;
+	const HILDA: u32 = 7;
+
+	#[test]
+	fn only_root_should_set_organizer() {
```

Comment:
Let's follow the same naming convention as the other tests
```suggestion
	fn set_organizer_should_only_accept_root_caller() {
```